packages/notices/src/store/actions.js

```diff
@@ -23,18 +26,43 @@ import { DEFAULT_CONTEXT } from './constants';
  * @param {?boolean}               options.isDismissible Whether the notice can
  *                                                       be dismissed by user.
  *                                                       Defaults to `true`.
+ * @param {?boolean}               options.speak         Whether the notice
+ *                                                       content should be
+ *                                                       announced to screen
+ *                                                       readers. Defaults to
+ *                                                       `true`.
  * @param {?Array<WPNoticeAction>} options.actions       User actions to be
  *                                                       presented with notice.
  */
-export function* createNotice( status = 'info', content, options = {} ) {
+export function* createNotice( statusOrNotice = DEFAULT_STATUS, contentOrOptions, options = {} ) {
```

Comment:
Minor: I personally don't like these mixed types arguments, I prefer separate actions creators calling each other.

Reply:
Yeah, I felt some of the revisions here were a little less than ideal, stemming from:

- In retrospect, maybe it would have been better to have the notice object be the sole / first argument, not `status, content`
- Supporting "raw HTML" as an alternative form of content

To the latter point, I could maybe imagine a case where we don't need to change anything here, but otherwise mark `content` as being raw HTML. Perhaps an option? Or something gross like `content = Object.assign( String( '<p>raw</p>' ), { __unstableRawHTML: true } );` (where gross is okay since it's unstable).

Reply:
I'm going to change this to limit the API impact, in using an option to define the content as being raw HTML.